README.md

```diff
@@ -1 +1,17 @@
-# esp-rom-elfs
+# ESP32 series ROM ELF files
+
+The [Releases](https://github.com/espressif/esp-rom-elfs/releases) page of this repository is used to publish the ELF files of ROMs used by ESP32 series of chips.
+
+The releases will be updated whenever a new Espressif chip or a chip revision is released.
+
```

Comment:
I'm editing IDF's docs and think we need to say a few words about how to choose the right ROM ELF revision:

```suggestion
## Choosing ROM ELF for a chip

In the release archives, you can find files with two filename patterns:

1. <CHIP_NAME>_rom.elf
2. <CHIP_NAME>_rev<REV>_rom.elf

If you found a ROM ELF with the first pattern, just use it.
Otherwise, find out your chip revision and use ROM ELF file with <REV> equals chip's revision. Or if the file does not exist use the last <REV> lower than the chip's revision. E.g.: for `esp32 rev2` use  `esp32_rev0_rom.elf`.
```